core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.html

```diff
@@ -107,17 +107,17 @@ <h2 class="stat-value-with-rating e2e-test-oppia-total-plays">
           </div>
           <div class="topic-selector-container">
             <oppia-topic-filter
-              *ngIf="(activeTab === TAB_NAME_TRANSLATION_SUBMITTER) || (activeTab === TAB_NAME_QUESTION_SUBMITTER)"
+              *ngIf="topicsAreFetched === 'true' && (activeTab === TAB_NAME_TRANSLATION_SUBMITTER) || (activeTab === TAB_NAME_QUESTION_SUBMITTER)"
               [(defaultTopicNames)]="allTopicNames"
               [(selectedTopicNames)]="selectedTopicNames"
               (selectionsChange)="applyTopicFilter()">
             </oppia-topic-filter>
           </div>
         </div>
       </div>
-      <contributor-admin-stats-table [activeTab]="activeTab" [filter]="filter"></contributor-admin-stats-table>
+      <contributor-admin-stats-table [inputs]="{ activeTab: activeTab, filter: filter }"></contributor-admin-stats-table>
```

Comment:
I'm pretty sure that `{ activeTab, filter }` will work due to JS Object syntax. I'd prefer writing it that way, since there's no code duplication.

Same can be said with line 193 of this file.

Reply:
extra code removed

Reply:
Alright. Keep it that way then.